docs/fsharp/style-guide/formatting.md

```diff
@@ -900,18 +900,44 @@ For lambda expressions, you may also want to consider placing the body of a lamb
 let printListWithOffset a list1 =
     List.iter
         (fun elem ->
-            printfn $"%d{a + elem}")
+             printfn $"A very long line to format the value: %d{a + elem}")
         list1
 
 let printListWithOffsetPiped a list1 =
     list1
     |> List.iter
-        (fun elem ->
-            printfn $"%d{a + elem}")
+           (fun elem ->
+                printfn $"A very long line to format the value: %d{a + elem}")
 ```
 
 If the body of a lambda expression is multiple lines long, you should consider refactoring it into a locally-scoped function.
 
+The parameters should always be indented relative to the function or `fun`/`function` keyword, regardless of the context in which the function appears:
```

Comment:
```suggestion
The parameters should generally be indented relative to the function or `fun`/`function` keyword, regardless of the context in which the function appears:
```

Reply:
Let's try to be a little softer on the language here.

Reply:
Done!